lib/src/actions.dart

```diff
@@ -69,14 +71,24 @@ class CustomSlidableAction extends StatelessWidget {
   /// Typically the action's icon or label.
   final Widget child;
 
+  /// {@template slidable.actions.shape}
+  /// [OutlinedBorder] of action, if is null, then [RoundedRectangleBorder]
+  /// used by default
+  /// {@endtemplate}
+  final OutlinedBorder shape;
```

Comment:
We should make `OutlinedBorder` nullable then.

Reply:
fixed